src/Foundatio.Jobs.Commands/Foundatio.Jobs.Commands.csproj

```diff
@@ -4,6 +4,11 @@
     <TargetFrameworks>net46</TargetFrameworks>
     <PackageTags>Distributed;Jobs;NETSTANDARD;Core</PackageTags>
   </PropertyGroup>
+  <PropertyGroup Label="Signing">
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <AssemblyOriginatorKeyFile>..\Exceptionless.snk</AssemblyOriginatorKeyFile>
```

Comment:
I'm thinking this wasn't copied or added because of a git ignore.. You might have to do a `git add PATHTOSNK -f`